### PR TITLE
Add tests for return value of DOMTokenList's replace() method

### PR DESCRIPTION
**Pull request by Zirro**
This adds tests for a boolean value being returned from the `replace()` method, as discussed in https://github.com/whatwg/dom/issues/577 and https://github.com/whatwg/dom/pull/582.

It also adds a test which attempts to replace a token that isn't present with one that is, since this case was not covered by the existing tests.

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25598)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/350721460)

**Review by foolip — DISMISSED** [review]
In the diff it looks like you've changed the indentation of many lines without changing anything else. Can you revert those changes to just have a minimal diff?

**Comment by Zirro** [comment]
@foolip I think you're referring to the lines where I've added an additional `null` argument. I'm not sure why they're not being highlighted like the lines further down.

**Comment by foolip** [comment]
@Zirro, you're right, I didn't see that. I'll remove by request for changes, but also not reviewing this right now.

File: interfaces/dom.idl [hunk 1]
```diff
@@ -548,7 +548,7 @@ interface DOMTokenList {
   [CEReactions] void add(DOMString... tokens);
   [CEReactions] void remove(DOMString... tokens);
   [CEReactions] boolean toggle(DOMString token, optional boolean force);
-  [CEReactions] void replace(DOMString token, DOMString newToken);
+  [CEReactions] boolean replace(DOMString token, DOMString newToken);
```
Comment (foolip) [inline interfaces/dom.idl]: @lukebjerring, this is a problem we're going to run in pretty quickly with generated IDL, where people update the tests together with the spec. As a reviewer, I think we should probably just say "yes please" and if there are any differences the auto-update will take care of it. WDYT?

**Review by annevk — APPROVED** [review]

File: interfaces/dom.idl (hunk 1 above)
Comment (lukebjerring) [reply interfaces/dom.idl]: Presumably, differences would be subtle (whitespace, indenting, etc). Until the updater is mature, and we document how to run it, will just have to accept changes :)

**Pull request merged**